src/main/java/core/basesyntax/stategy/handler/ReturnOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.stategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
According to the checklist, the class should be placed in the 'strategy' package, but the current package is named 'stategy', which seems to be a typo.